Fix #6062 Aligned the logos of sharing links in the bottom of a exploration page

JeeBoomBoi opened the pull request:
<!--
  - Thanks for submitting code to Oppia! Please fill out the following as part of
  - your pull request so we can review your code more easily.
  -->

## Explanation
This issue was due to a padding error in one of the css files. A change was made in the padding value so that the logos are aligned in a proper way and don't hide in the bottom of a browser page.

<!--
  - Explain what your PR does. If this PR fixes an existing bug, please include
  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
  - when this PR is merged.
  -->

## Checklist
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The linter/Karma presubmit checks have passed.
  - These should run automatically, but if not, you can manually trigger them locally using `python scripts/pre_commit_linter.py` and `bash scripts/run_frontend_tests.sh`.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR follows the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).
- [x] The PR is assigned to an appropriate reviewer.
  - If you're a new contributor, please ask on [Gitter](https://gitter.im/oppia/oppia-chat) for someone to assign a reviewer.
  - If you're not sure who the appropriate reviewer is, please assign to the issue's "owner" -- see the "talk-to" label on the issue.
Before change in code:
![patch12](https://user-images.githubusercontent.com/43813343/51608959-e4e29a00-1f3e-11e9-91bc-eb6c912fe0ec.jpg)

After change in code:
![patch1](https://user-images.githubusercontent.com/43813343/51608987-f62ba680-1f3e-11e9-9422-22d1acc8b3ee.jpg)

vojtechjelinek committed:
Fix part of #5002: Move ALLOW_YAML_FILE_UPLOAD from feconf to constants (#5989)

* Move ALLOW_YAML_FILE_UPLOAD from feconf to constants

apb7 committed:
Fixes #5839: Filter slow network logs for mobile tests on Browserstack. (#6090)

* Fixes #5839: Filter slow network logs for mobile tests on Browserstack.

* Test commit: Run only mobile tests on Travis

* Revert test commit and run all tests

brianrodri committed:
Fix part of #5591: Migrate Playthrough Feature Data into ExplorationFeaturesService (#6044)

* Fix hola's multiple choice question.

* better description

* expose playthrough_recording_enabled through features handler

* remove playthrough whitelist handler

* remove playthroughwhitelisthandler.

* move whitelist functionality to backend

* fix: no-multi-spaces

* import features  service

* import feature services

* remove unused dependency

* fix imports

* sort dependencies

* add is_ prefix

* remove unused dependency

* move features service to services directory

* move imports

* check for null

* import service

* assign to correct property

* remove unused dependency

* sort imports

* organize constants and setup

* move features handler to reader module

* fix import

* remove unused import

* add import

* use new description for search

* move feature handler to new features module

* minor nit fixes

* remove extra whitespaces

* reorganize setup

* fix unused-import

* fix imports

* use better names

* introduce contextmanager for properties

* revert changes

* add swap_property_value to test_utils

* add swap_property_value to test_utils

* Fix unused-import

* fix unused-import

* fix docstrings

* s/get_features/get_features_json

* improve variable name

* improve variable name

* allow fetching EXP_ID by default

* use self.EXP_ID where it improves readability

* plural improvement

* s/swap_property_value/swap_property_value_context

* explicitly name kwarg

* fix file docstring

* it's 2019 :)

* fix typo

* fix docstring

* better contextmanager organization

* make exp_id required

* Update core/templates/dev/head/services/PlaythroughServiceSpec.js

Co-Authored-By: brianrodri <brian@brianrodri.com>

* store URL in feconf

* better wording

* better var name

* better docstring

* just use config_domain directly

* better file docstring

* simpler code

* fix unused-import

* improve docstring

* fix typo

* organize docstring

* better yields description

* improve docstring

* Update test_utils.py

Improve description

* refactor playthroughservicespec

* remove extra closing brace

* minor fixes

* better it descriptions

* revert changes to swap docstring

* add missing newline

* move commiter_id to end of args

* use proper expect syntax

* strip 1

* add checks for more properties

* better it description

* add constant for the 'latest' schema version

* don't explicitly pass schema version

* test that schema changes are remembered

* use JSDoc notation for optional parameters

* only createNew should allow convinience optional param

* fix jsdoc

* use default value JSDoc syntax

* strip 1

* fix indent

* better constant name

* use set_config_property

* add test case

* check that schema version is valid

* append () to function tests

* revert changes

* fix brace-style

* use local ref to access laof

* revert description

* add docstrings

YashJipkate committed:
Fix #5923: Refactoring and clean up backend tests. (#6074)

anubhavsinha98 committed:
Fixes part of #5134 : Making the coverage of email_jobs_one_off to 100% . (#6026)

* Increased Coverage for core.domain.email_jobs_one_off

* Added yield statement in map only

* Changed the yield statement and function params

* Changed the yield statement and function params

* 100% Coverage

* 100% Coverage

sandeep-patel11 committed:
Fix #5729: Adds a numerical progress status in the translation tab. (#6097)

rafalk342 committed:
Fix #6078: Updating publish button after modal is dismissed. (#6096)

seanlip committed:
Update authors and changelog for v2.7.2. (#6109)

* Update authors and changelog for v2.7.2.

* Fix errors.

Commit:
Fix #6104: Use sub-process in pre_push_hook (#6114)

* Use subprocess

* Fix lint errors

abhaygupta97 committed:
Adding tests to increase coverage of OutcomeObjectFactory.js (#6098)

geetchoudhary committed:
Fix #5678: Add swipeable carousel to Library page in mobile view (#6087)

* adds swipeable carousel

* prevent swiping past first and last card

* fixed linting issues

* changed function logic

* made references from js file

* minor fixes

* minor nits

abhaygupta97 committed:
Fix part of #5134: Adding tests for core.domain.collection_jobs_one_off (#6061)

mighty-phoenix committed:
Fix #5771: Ensure that the active tab of translation tab is maintained everytime save draft/publish changes is clicked. (#6076)

* Fixed the problem of shifting of focus on publishing changes by removing necessary code that re-initialized translation tab everytime a change was made.

* Altered fix due to failing tests.

* Added e2e test to ensure the active sub tab doesn't change on save draft

* Fixed lint error

* Fixed style nits

* Removed accidentally left 'fit'

* CamelCase

* Line shortened

* lint fix

* Added more checks.

* Removed style nits

* fix

* fix nits

* fix nits

* grammar improved

* Changes

* Fit changed to it

* Fixed style nits

* Reverted

* Change made

* Some more changes

* Final changes

* Style nits fixed

* Style nits fixed

* Style nits fixed

* Style nits fixed

* Style nits fixed

* final style nit

* final style nit

geetchoudhary committed:
fix tooltip placement (#6119)

ankita240796 committed:
Fix #6089: Moved common css to oppia.css and removed extra comments for sync (#6107)

geetchoudhary committed:
Fix #6118: Improved spacing between reload buttons on admin page (#6123)

ankita240796 committed:
Fix #3074: Added height limit warning for card content (#5997)

keshavg2 committed:
Fix #4833: prevents text overflow  in answer statistics page (#6137)

* prevents overflow text in answer statistics

* prevents text overflow in answer statistics page

* Update frequency_table_directive.html

ctao5660 committed:
Fixes #4407 KaTeX text no longer goes outside text container in MathExpressionInput (#6130)

* KaTeX text no longer goes outside text container

* Fixed buggy clicks

* addressed comments

mighty-phoenix committed:
Fix #6122: Add $log service to prevent the error arising on rejecting suggestions (#6124)

JeeBoomBoi committed:
fix #6062 aligned the logos of google+, facebook and twitter on any page of exploration

keshavg2 commented:
hi @jeeb0xical can you please write in the explanation
- Fixes #bugnum , followed by a short summry of the changes
 Thanks

JeeBoomBoi commented:
Hi @17bcs029, I made the required changes 
thanks !

keshavg2 commented:
hi @jeeb0xical actually I'm tell that written under the explanation.
Thanks

JeeBoomBoi commented:
Done @17bcs029. Sorry for that. This is my first pull request

keshavg2 commented:
@jeeb0xical It's okay and also best of luck for your first pull request.

DubeySandeep commented:
@jeeb0xical, Thanks for the PR! Can you please attach a screenshot of the UI changes? 

@anubhavsinha98, can you please help us doing UI review for this PR?

JeeBoomBoi commented:
@17bcs029 Thanks !!

JeeBoomBoi commented:
@DubeySandeep I have attached a image.

Dilshaad21 commented:
Maybe @jeeb0xical has not aligned the last logo correctly w.r.t the other logos.I would suggest @DubeySandeep to go for my PR   @anubhavsinha98 .

JeeBoomBoi commented:
@Dilshaad21 The fix says to align the logos of google+, facebook and twitter only. The google classroom should be above compared to the other logos.

Dilshaad21 commented:
I haven't ever thought that I would begin my first opensource contribution and would be welcomed by  great people like you @jeeb0xical!

anubhavsinha98 reviewed (commented):
Hi @jeeb0xical can you upload the screenshot of the corrected alignment of the logos ?

JeeBoomBoi commented:
@anubhavsinha98 The pic I have uploaded in the explanation is the corrected logo. I have a doubt, Should the google classroom logo should be aligned with others.

Dilshaad21 commented:
@jeeb0xical I can help you set the alignment of the last logo.

JeeBoomBoi commented:
@Dilshaad21 I would like to receive help but I just want to confirm is that what is expected from the PR. Thanks!

JeeBoomBoi commented:
![patch1-2](https://user-images.githubusercontent.com/43813343/51610071-d053d100-1f41-11e9-9c05-17ac4397104b.jpg)
 
Should the icons look like this or like the one in the explanation of my PR. @anubhavsinha98

anubhavsinha98 commented:
> ![patch1-2](https://user-images.githubusercontent.com/43813343/51610071-d053d100-1f41-11e9-9c05-17ac4397104b.jpg)
> 
> Should the icons look like this or like the one in the explanation of my PR. @anubhavsinha98

It should look like this.

JeeBoomBoi committed:
final changes made in patch1

JeeBoomBoi commented:
Done the changes now. I'm pretty sure this will sort out this issue @anubhavsinha98. I hope my PR gets merged. Thanks !

anubhavsinha98 commented:
Can you upload the latest screenshot of UI ?

JeeBoomBoi commented:
> ![patch1-2](https://user-images.githubusercontent.com/43813343/51610071-d053d100-1f41-11e9-9c05-17ac4397104b.jpg)
> 
> Should the icons look like this or like the one in the explanation of my PR. @anubhavsinha98

It looks like this now @anubhavsinha98. I made changes and took a screenshot.

anubhavsinha98 commented:
One thing to ask @jeeb0xical , which exploration did you loaded to check this ?

JeeBoomBoi commented:
@anubhavsinha98 I wasn't able to replicate this in a localhost. I made changes using console and it worked out.

anubhavsinha98 committed:
Fixes #5371: One Off Job to populate message_count in GeneralFeedbackThreadModel. (#5999)

* earlier commit

* Populate message_count using MR job

* Populate message_count using MR job

* Populate message_count using MR job

* Populate message_count using MR job

* Populate message_count using MR job

* Populate message_count using one off mr job

* Populate message_count using one off mr job

* made message_count required

* Populate message_count field one off job

* Populate message_count field one off job

* Populate message_count field one off job

* Added assertion

* Added pre job check

* Edited One Off Job

* Edited yield statement

* Job is working

* Functions removed

* Made default=0

* Removed Import

* Imrpoved

* Job completed

* Change Output

* Change Output

* Added NO-OP test case

* Remove pylint disable

anubhavsinha98 commented:
@DubeySandeep , I think this issue can't be replicated in the localhost. Did you know any way how to get this issue in the localhost, I tried adding all the explorations but I was not able to find this issue ?

JeeBoomBoi commented:
Yes @anubhavsinha98. I tried everything but I wasn't able to implement those icons.

seanlip committed:
Routine update to translations. (#6164)

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Add Hungarian to list of supported languages.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix broken tests.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix translation error.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix translation error.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix errors in translations found by backend tests.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix broken tests.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix errors in translations.

* Fix a Python linting issue.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix spelling error in description

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix backend tests.

ctao5660 committed:
fixed text predictor test (#6120)

DubeySandeep commented:
@anubhavsinha98, have you tried loading demo collections from admin page? FYI, we have removed the G+ share icon recently. Also, you can check the wiki page for "How to create a collection". Hope this helps!

Dilshaad21 commented:
Here! #6145 
![ubuntu 64-bit - vmware workstation 21-01-2019 18_29_45](https://user-images.githubusercontent.com/45489945/51898254-89992780-23d6-11e9-9ca7-28267d6dd502.png)

brianrodri committed:
Fix #5975: Create One-off Job to Remove Invalid Playthroughs (#6131)

* Fix hola's multiple choice question.

* override delete to recursively delete playthroughs

* initial commit of one-off job

* initial commit of tests

* use correct import

* fix undefined-variable

* fix bad-reversed-sequence

* refactor predicate

* fix bad-super-call

* fix bad-reversed-sequence

* fix redefined-outer-name

* add punctuation

* keep statements closer

* improved structure

* add docstring, better var names

* don't use id

* remove unused import

* just use exp_id

* remove delete override

* positive check first

* clean up line-too-long errors with consolidated code

* capitalize exp_id class constants

* capitalize class constant usages

* use state_name

* remove wrong variable usages

* only use two-lines to delimit classes

* consistent usage of ExplorationChange

* use correct property

* pass models not ids

* ensure  playthrough_model exists

* pass real models not ids

* add test for removing non-whitelisted explorations

* use  datetime.datetime

* finish tests

* use assertRaisesRegexp

* remove unused var

* make public

* clean up playthroughs after bad ones are removed

* add comment

* gather related logic

* better names

* simplify class

* singular test class name

* pluralize to silence docstring error

* factor out common code

* better test case name

* better naming

* choose different date

* use comment with punctuatioin

* use comment with punctuatioin

* assign to temp variable

* move setUp to beginning of class

* add check that job is removed from queue

* remove verbose constant

* clearer names

* prettier spacing

* minor spacing fix

* minor fixes to docs

* better test case names

* proper docstring

* clearer doc

* add emphasis

* add test case with many unresolved issues

* organize assertions

* fix test case name

* add extra newline before class

* better comment

* fix doc string

* fix line-too-long

* better naming

* s/it's/its

* remove new

* better wording

* punctuation

* better spacing

* better var name

* better naming

* better tests

* better comment

* better temp var name

* use temp var to improve readability

* better comment

* remove docstring

* prioritize deletioni

* add whitespace for clearer separation

* fix typo

* separate final model deletion logic

* improve comment/var naming

* pop inplace

* do more logic in for loop

* use generator comprehension

* move playthroughs_deleted var closer to usage

* more efficient checks

* update playthrough_ids

* assign directly to dict

* comment instead of new line

* improve comment

* better comment

* organization changes

* fix typo

* consistent list literals

* use kwargs for readability

* share run_one_off_job with other classes.

* consistent usage of spaces in dict literals

* consistency improvements

* consistency

* fix indent

* simplify logic by copying list before iterating

* improve  comment

* check jobs in queue is 0 before enqueueing

* fix GSoC release date

* improved comment

* use slice assignment

* improve comment

* improve comment

* improve comment

* consolidate logic

* consolidate variables

* prefer index to i

* just use list comprehension

* let code speak for itself

* let code speak for itself

* add comments explaining slice assignment

* comment on slice assignment

* push cut-off date to september 1st

* initial commit of playthrough audit job

* fix whitespace before comma

* add error key for invalid playthroughs

* add one-off-jobs to registry

* fix errors

* better organization

* better comments

* Consistent yield result: 1-tuple

* s/invalid/illegal

* better wording

* specify who defines which explorations are safe

* remove extra newline

* add indent

* add punctuation to error strings

* better docstring

* remove unused constant

* shorter descrpition

* actual map description

* make description accurate

* use simpler param name

* fix wording of timing

* stronger wording

* stronger wording

* use better heading description

* s/invalid/illegal

* consolidate lines

* remove extra space

* explicitly call validate

* use None for validate error

* collapse branch

* use positive check

* clarify 'all models'

* cleaner code organization

* gather related logic

* improve docstrings

* remove _ =

* simpler returns doc

* use bullets

* improve readability

* s/Remove/Delete

* distinguish datetime

* update registry

* add get_task_response method

* allow multiple values

* better invalid schema violation

* better key name

* simplify docstring

* fix line-too-long

* store variable only once

* s/value/audit_data

* return output from run_one_off_job method

* add test case for good playthrough

* fix line-too-long

* improve test case names

geetchoudhary committed:
Fix #5019 : The continue button is now visible with long feedback (#6136)

* add scroll feature

* changed max height

* added padding for mozilla

* added overflow instead of padding

* reduced max-height

* fixed height for smaller screens

aks681 committed:
UI changes to topic and skill editor (#6091)

* Finished 3 cells of schedule doc

* Done with all the changes

* fix lint

* made initial review changes

* created a new questions-list directive

* minor change

* added skill id fields to story editor

* lint fix

* made review changes

* changes to function names

* Made review changes

* fixed test

nithusha21 committed:
Reorder e2e test suites into more logical categorization (#6036)

* WIP

* refactor exploration editor/player test suites

* linting fixes

* remove extra file

* split exploration tabs into separate test files

* change configuration file to remove old files, and include new ones

* alphabetize, and changes to .travis.yml

* change year to 2019

* fix missing imports in test suites

* fix stray space

* disable travis test for blank test file

* delete unused suite

vibhor98 committed:
Fix #5822: Add customization option to allow multiple sort items in the same position in Drag and drop interaction. (#6177)

* Add customization arg.

* Perform exploration schema upgrade.

* Fix backend tests.

* Correct schema version.

* Modify schema version in question editors.

* Fix version issue.

* Fix lint issues.

* Fix backend tests.

* Modify zip file.

mighty-phoenix committed:
Fix #5745: Redesign audio bar of the translation tab. (#6167)

bansalnitish committed:
Changed word-break to word-wrap (#6189)

nithusha21 committed:
Fix check for clean working tree in common.py (#6191)

* change wording of check for clean directory

* keep old message for compatibility

mzaman07 committed:
Fix #5572: added translation help explaining translation features (#5796)

* Fix for issue 5572 adding translation help.

* fixed all lint problems

* change the html back

* fixed forgotten html change

* added in-memory database reference for the one time tutorial and its read write operations.

* added request methods for setting and adding translation tutorial state for session activity based. Also added initialization data settings for initial loads.

* changed the translation tour to match the reviewed changes. Added a service to do a firsttimeonly tour based on user sessions.

* added a way to review the translation tutorial after the first time.

* modified tests to handle the new translation tutorial section.

* lint fix

* Fixed lint error missing space on infix operatior : in LogicProof.js

* lint match space and line match

* reorder dependencies for EditorNavigtionDirective

* changed response content-type for first time translation tutorial

* added test for the traslation first time tutorial for http methods

* first time translation tutorial storage test added

* lint fixed

* lint fix

* more lint fixes

* refixed the merge conflict in this file

* fixed e2e tests for translation tutorial

* added e2e test for the translation tutorial.

* fixed lint

* fixed test for lint

* unified state tutorial services editor and translation.

* lint end line for StateTutorialFirstTimeService

* lint-fix

* fixed for test

* modified tutorial content with reviewed changes.

* lint fixed

* fixed based on review for code readibility, not 100% review coverage.

* moved translation tutorial and tests from editor to translator

* changed name of imageurl from editor to translation for translation tutorial

* added exit tutorial to test of sub-tab in translation tab.

* updated doc string

* minor fixes for review

seanlip committed:
Fix Content-Disposition headers being unicode instead of str. (#6192)

DubeySandeep committed:
Fix #4278:  Re-introduce 3-letter language codes for explorations and collections. (#6182)

* Re-introduced 3-letter langauge code for explration and collections.

* Undo some changes.

* Added review changes.

DubeySandeep committed:
Translation Script: Adds TranslationScript and ContentTranslation object in state domain. (#6121)

* Adds TranslationScript domain object with tests.

* Added ContentTranscriptions object in state and added test for the same.

* Fixed some style issues.

* Added more validation check.

* Fixed some lint issues.

* Added new methods as for requirments.

* Added review changes.

* Added review changes and moved 3-letter check in utils.

* Reverted changes made for language_code./

anubhavsinha98 committed:
Fixes #5508: Feedback threads in feedback tab reorder now in real time. (#6183)

* Reload Option

* Intital

* Testing

* Fixed Feedback Ordering in real time

* fetch only when updated

* Fixed linting issues

* Updated the var

* Changed var name

* Fixed the changes

* Fixed linting issues

* If statement replaced

* Fixed linitng

brianrodri committed:
Add None/deleted checks to DeleteIllegalPlaythroughsOneOffJob (#6195)

* Fix hola's multiple choice question.

* add None and deleted checks

DubeySandeep committed:
Adds functionality to internally manage to add and delete content ids into assets dict. (#6171)

mjprince committed:
Fix part of #4231 Make the add response modal scroll visible at all times, via css (#6116)

* Providing a temporal solution to the: Add Response tab in creator view can not be scrolled down bug

* Making the vertical overflow automatic

* Only making the choices scroll automatically

* Put scroll only on choices

* Put scroll only on choices

seanlip committed:
Remove extraneous file. (#6196)

vojtechjelinek committed:
Update app.yaml (#6184)

* Update app.yaml

* separate handlers into sections

* Change DEV_MODE to true

* Changes to app.yaml

Dilshaad21 commented:
@anubhavsinha98 if you give permission I can open my PR as no one is coming forward for the issue.Thank you.

Dharmesh-Poddar committed:
fix #6140 learner answer overflow (#6154)

Dharmesh-Poddar committed:
alignment of material icons (#6147)

vojtechjelinek committed:
Remove Open Sans and Rubik fonts (#6208)

ulloaluis committed:
Fix part of #5134: Increased coverage to 100% for mailgun_email_services.py (#6133)

* Fix #5134: Improve coverage to 100%

* Fix #5134: Improve coverage to 100%

* Fix #5134: Improve coverage to 100%.

* Fix #5134: Improve coverage to 100%.

* Fix #5134: Improve coverage to 100%.

* Fix part of #5134: Increased coverage to 100% for mailgun_email_services.py

* Fix part of #5134: Increased coverage to 100% for mailgun_email_services.py

* Fix part of #5134: Increased coverage to 100% for mailgun_email_services.py

vojtechjelinek committed:
Add aplication_readable to directive htmls handler (#6216)

aks681 committed:
Added topics and skills dashboard e2e tests (#6204)

* added topics and skills dashboard e2e tests

* fix lint

* fixed test

* added topic and skill deletion tests

* minor change

* made review changes

DubeySandeep commented:
@anubhavsinha98 & @jeeb0xical any updates on this PR?

brianrodri committed:
Rename ExplorationIssue* to PlaythroughIssue* (#6198)

JeeBoomBoi commented:
@DubeySandeep I'm working on it. Currently, I have exams in my college. Once exams are over I will work in a fully fledged manner.

brianrodri committed:
Revert "Rename ExplorationIssue* to PlaythroughIssue* (#6198)" (#6220)

This reverts commit 0a37edf511a52619201109c4802314921d3f3d72.

ankita240796 committed:
Fix part of #5134: Added test for exp one off jobs (#5972)

* Added test for exp one off jobs

* Fix minor nit

* Add test for image and audio file jobs

* Fix misleading comment

* Add more test to increase coverage

* Fix comments

* Improve consistency

* Fix minor nit

* Address review comments

* Remove pylint pragmas

* Add setup function

* Add test for multiple exps

* Add function to run job for deleted exp

* fix backend test

* Fix test

* Address review comments

* Fix comments

* Update comments, test job count

YashJipkate committed:
Fix part of #5939: Custom pylint extension for checking single character files and newline characters. (#6172)

* Custom pylint for newline at EOF

* Renamed the class and modified docstring

* Minor nits

* Address Review

* Added tests for the added extension

* Added periods as asked in review

* Updated node names as requested in review.

* Updated other node names too

Commit:
Fix #6103: Add summary of errors for lint checks on Travis (#6165)

* Add summary of of all errors

* Fix error count

* Restored previous file

* use context managers

* Move pre_commit_linter to top

* delete log files

* Add message to summary section

* Print validating statements before summary section

* Remove linting statements from summary

anubhavsinha98 committed:
Updated the topic backend handler (#6185)

* Update topic handler

* Update

* :

* Added unit test

* Added Unit Test

* Fixed Linting

* Fixed Linting

* Added Unit Test :: Completed

* Added uncategorized skill ids

* Fixed docstrings

ankita240796 committed:
Fix part of #5134: Added test for core.storage.file.gae_models (#5960)

* Add test for gae_models

* Addressed review comments

* Improve naming, remove extra lines

* Improve function naming and split test, remove protected access test

* Update get version test to check if content is reconstituted correctly

* Fix variable and function names

* Add test to check get version for an absent version

* Add test for compute snapshot

* Address review comments

* Remove version comparison

* Add version to reconstituted model, add test for same

* Fix failing admin test

DubeySandeep committed:
Added active content id service to fix translation tab issues. (#6211)

* Added new service to fix translation tab issues.

* Fixed inavlid var name.

* Addressed review comments.

* Added correct year for new files.

mighty-phoenix committed:
Fix#6188: Fix word-break in explorations (#6187)

* Fix word-break in explorations

* Fixed word-break in all occurences

* Fixed all instances of word-break

* Final Changes

* Changed

* Added css styling with class.

* Fixed all occurences of word break and tested.

* word-break: break-all removed from all places. No occurence of it in our codebase now.

ujjwalgulecha committed:
Fix part of #5134: Increased coverage to 100% for question_editor (#6217)

* Fix part of #5134: Increased coverage to 100% for question_editor

* refactored all tests in the class

* minor refactor

brianrodri committed:
Improvements to PlaythroughAudit (#6239)

* Fix hola's multiple choice question.

* add reference data to audit

* add create exp issues helper

* remove delete job from registry

* add self to arg list

* use generalized exception

* add reference tests

* remove check for non-existing exp_issues model

* add docstring

* pluralize set

* no punctuation in error message

* alphabetize classes

ankita240796 committed:
Fix #5487: Fix load issue with multiple same images (#6170)

varun-tandon committed:
Fix #5134: Improve coverage to 89% by implementing anonymous feedback (#6086)

* Fix #5134: Improve coverage to 89% by implementing anonymous feedback

* Updated code to match upstream

* Fix #5134: Improve coverage to 89% by implementing anonymous feedback

* Fix #5134: Improve coverage to 89% by implementing anonymous feedback

* Fix #5134: 89% coverage core.controllers.learner_dashboard

* Update learner_dashboard_test.py

JeeBoomBoi committed:
verified in localhost

JeeBoomBoi commented:
Verified it in local host @anubhavsinha98.  
![screenshot from 2019-02-11 18-33-05](https://user-images.githubusercontent.com/43813343/52565265-5e79f380-2e2c-11e9-9731-d55a1d109ed3.png)

anubhavsinha98 reviewed (approved):
LGTM!

anubhavsinha98 commented:
Hi @DubeySandeep can you please review?

DubeySandeep commented:
@anubhavsinha98, wanted to know whether you have tested this where ever sharing-links directive is used? (Like exploration editor, exploration player, collection player page, creator dashboard page.)   
![image](https://user-images.githubusercontent.com/16653571/52571206-1c0be300-2e3b-11e9-8f62-d83384cf34ec.png)

YashJipkate committed:
Fix #5465: Added check for prohibited imports (#6180)

* Added check for prohibited import (storage layer importing domain layer)

* Added tests

* Ignored check in places already using it and renamed msgid

* Test

* Test

* Nits

* Updated node types in docstrings

* Ignored check in test files

* Added check prohibiting domain layer importing controller layer

* Modified tests.

* Modified error message.

anubhavsinha98 commented:
Hi @DubeySandeep, I was able to check for 2 instances, one when we play the test collection(in footer) and one during the publishing of the exploration(in the card). But was not able to reproduce on creator_dashboard.html.

DubeySandeep commented:
@anubhavsinha98, What about "info card modal"? Also, have you tested this on other browsers?

> But was not able to reproduce on creator_dashboard.html.

I'll do this. Thanks for the info. :)

anubhavsinha98 commented:
> @anubhavsinha98, What about "info card modal"? Also, have you tested this on other browsers?
> 
> > But was not able to reproduce on creator_dashboard.html.
> 
> I'll do this. Thanks for the info. :)

@DubeySandeep 
I have checked for info card modal also. All the 3 instances are working fine in both chrome and firefox.

abhaygupta97 committed:
Fix part of #5002: Remove promo bar from globals (#6169)

* Fix part of #5002: Remove promo bar from globals

* Making requested changes

* Minor fixes

* Rename ResourceService to PromoBarService

* Bug fix

* Implementing proposed changes

* Making minor changes

* Removing else statement

* Adding resolve in PromoBarService

ankita240796 committed:
Refactor for suggestion modal display  (#6093)

* Added new service for suggestion modal display

* Fix nits in develop

* Fix structure issues

* Fix minor nit

* Change 2018 to 2019, include scripts in correct file

* Add service to handle common code for suggestion modal display

ankita240796 committed:
Fix #6251: Add basic typescript configuration (#6256)

* Change way of parsing comments

* Add types install

* Add tsconfig

* Add typescript installation

* Remove comment formatting to avoid duplicate identifier error

* Improve naming

* Fix failing test

* alphabetize items

* fix space

* Add dummy constants for testing comment removal

YashJipkate committed:
Fix #6128: Successfully load mobile view in '/parents' page. (#6259)

* Fixed mobile view of stewards page

* A better way to do this

* Removed unnecessary changes

* Address review and fix hidden overflow

* Removed extra semicolon.

* Accomodated some inlines to class

abhaygupta97 committed:
Fixes #6244: Fixing the cancel button in hint editor (#6250)

* Fixes #6244: Fixing the cancel button in hint editor

* Making requested changes

DubeySandeep committed:
One off job to validate exploration migration can be carried out successfully. (#6249)

DubeySandeep committed:
Adds code-owners for teams/projects. (#6266)

* Codeowner structure changes.

* Fixed typos.

* Added missing codeowners for suggestion project.

* Fixed typos.

* Added review changes.

* Added comment for explanation.

* Fixed typos.

ankita240796 committed:
Fix part of #6254: Refactor to implement rule of 1 (#6267)

* Refactor for end exploration interaction

* Refactor for logic proof interaction

* Refactor for multiple choice input interaction

* Refactor for numeric input interaction

* Refactor for pencil code editor interaction

* Refactor for set input interaction

* Refactor for text input interaction

* refract directive files of interactions CodeReplt, FractionInput, ImageClickInput

* Refactor for continue interaction

* Refactor for drag and drop sort interaction

* Refactor for graph input interaction

* Refactor for item selection interaction

* Refactor for math expression input interaction

* Refactor for number with units interaction

* Update test, fix mistakes

* Refactor for interactive map interaction

* Fix names and comments

* Fix indent

anubhavsinha98 committed:
Added story viewer backend handler  (#6237)

* First

* First

* Second

* Third

* Fourth

* Added new file

* Added test file and changes

* Added test file and running

* Fixed linting issues

* Fixed linting issues

* Fixed linting issues

* running tests success

* merge conflicts removed

* Fixed

* Fixed

* Changes_Done

* Changes_Done

nithusha21 committed:
Update changelog, authors and contributors for v2.7.3 (#6273)

* Update changelog, authors and contributors for v2.7.3

* remove stray <

anubhavsinha98 committed:
Fixes part of #6233 : Improved the error message for match line breaks controller (#6246)

* Improved the error message for match line breaks controller

* Updated the msg

ankita240796 committed:
Fix #6252: Add a script to change file extensions (#6260)

* Add script to rename files

* Add test

* fix imports, use tempfile to create dir

* change ext to extensions

* Move changing extensions from setup to test

DubeySandeep committed:
Fix CODEOWNER wrong dir issue. (#6277)

* Fixed dir location issues.

* Fix some more issues.

seanlip committed:
Fix 'Promise is undefined' server error. (#6281)

seanlip committed:
Add some additional pre-release checks. (#6284)

seanlip committed:
Delete old one-off jobs. (#6283)

DubeySandeep committed:
Adds codeowners for other projects. (#6287)

anubhavsinha98 commented:
Hi @DubeySandeep, any updates here?

kevinlee12 committed:
Change email for Kevin Lee in AUTHORS and CONTRIBUTORS files (#6293)

DubeySandeep committed:
Translation Script: Exploration migration including services for handling translation script.  (#6175)

anubhavsinha98 committed:
Fixes #6282: Run front-end tests in pre-push hook, only if changes are made in JS files. (#6289)

* First

* First

* Second

* Third

* Fourth

* Testing pre_push_hook

* Added pre_push_hook feature

* Added pre_push_hook feature

* Added pre_push_hook feature

* Fixed linting

* Fixed linting

* Fixed linting

* Removed Excluded

* Removed Excluded

* Removed Excluded

* Changed func name

* Changed func name

* Changes Done

* Added Args & Return

* Changed Docstring

varun-tandon committed:
Fix part of #5134: 100% coverage for core.platform.models (#6263)

seanlip committed:
Update deployment script to check for indexes in advance. (#6300)

* Update deployment script to check for indexes in advance.

* Address review comments.

seanlip committed:
Routine update of translations (#6302)

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Add Hungarian to list of supported languages.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix broken tests.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix translation error.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix translation error.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix errors in translations found by backend tests.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix broken tests.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix errors in translations.

* Fix a Python linting issue.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix spelling error in description

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Fix backend tests.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

* Localisation updates from https://translatewiki.net.

Commit:
Fix part of #5476: Add checks to detect spaces in docstring (#6231)

abhaygupta97 committed:
Removing scripts and include statements for popovers (#6270)

* Removing scripts and include statements for popups

* Making requested changes

* Adding documentation

* Making minor fixes

* Moving html to separate file

* Removing use of templateCache

* Making minor changes

* Making suggested changes

anubhavsinha98 committed:
Fix #6004: Added a presubmit check to ensure imports of JS files are sorted (#6125)

* First

* First

* Second

* Third

* Fourth

* Fixed imports order

* Few more

* 1800

* 1900

* Removed

* corrected

* 2010

* Final

* resolved linting issues

* resolved linting issues

* Fixed HintsAndSolution

* Fixed Hintandsolution

* Fixed ExplorationCreationService.js

* Added presubmit check

* Fixed all files

* Added one time parsing only

* Made some changes

* Update pre_commit_linter.py

* changes done-linitng issues fixed

* Fixed the changes

* Updated the changes

* Changes done

* Fixed linting

* Fixed failed travis ci check

* Fixed changes

* Fixed changes

* Fixed linting issues

* Changes done

* Updated LocalNav.js

ujjwalgulecha committed:
Fix #4057 Make PlayerPositionService.js 100% test coverage (#6258)

* Fix #4057 Make PlayerPositionService.js 100% test coverage

* address comments

* minor refactor

* minor refactor

BenHenning committed:
Correct comment string in UserSubscriptionsModel (#6310)

The original string seemed backwards based on how this is used (see https://github.com/oppia/oppia/blob/0d676f9c73d16639b05273e61be683e19fd604e2/core/domain/subscription_services.py#L145). It appears that this field, as expected, is actually storing the IDs of creators this user is subscribed to.

nithusha21 committed:
Add issue template for server errors (#6308)

* add issue template for server errors

* improve about section

* review changes

* minor nits

seanlip committed:
Add a script for updating indexes. Move gcloud functions into a separate module. (#6305)

* Add a script for updating indexes. Move gcloud functions into a separate file.

* Further changes needed to use gcloud for deployment.

* Remove unused code.

* Add ability to specify custom versions.

* Do not switch to new version after uploading.

* Update fileoverview docstring.

bansalnitish committed:
Fix #2306: Fixes signup issue when a new tab is opened (#6235)

* Fix signup issue

* Fix failing test

* modal to signup and added test

* removed useless variable

* Add docstrings and modify function names

* Addressed review comments

* Replace post with post_json

* Addressed review comments

* Fixed lint errors

* Addressed review comments

apb7 committed:
Port back-end tests from Travis to CircleCI (#6234)

* Basic config

* Minor nit

* Test caching mechanism

* Try integration of frontend and backend tests

* Syntax fix

* Add defaults back to each job

* Add checkout for each job

* Add checkout to setup job

* Add anchor for restore_cache

* Add e2e tests accessiblity

* Add third_party_cache and setup_cache key

* Fix cache paths

* Change working dir path

* Use abs paths for cache

* Try specifying chrome version

* Add libappindicator

* Tweak

* Try fixing path and run only e2e tests

* Add lint, frontend and backend tests to config.yml

* Remove lint, frontend and backend tests from travis.yml

* Remove undefined anchor

* Fix yml

* Install dependencies

* Minor tweaks

* Move init hook to Master section in pylintrc

* Test commit: try changing the working dir

* Port backend tests from Travis to CircleCi

* Revert changes made in pylintrc

* Revert changes to Travis and comment out backend and frontend tests' run

* Run frontend and generate coverage

* Install codecov as a dependency

* Add sudo to overcome permission error

* Re-order codecov install

* Add branches to be build and try enabling backend test coverage

* Fix config

* Remove branch section

* Try: lint tests

* Try: lint tests

* Try: Add dependency installation to lint tests job

* Try: Caching

* Try: Caching

* Try: Caching

* Try: Caching

* Try: Caching

* Try: Caching

* Run lint, frontend and backend tests

* Generate date for each job

* Comment out lint test and re-order third party installation

* Fix Travis

* Remove --generate_coverage_report flag from backend tests job

* Add explanation to travis config for commenting checks out

* Re-order dependency install

* Re-order dependency

* Revert: Remove --generate_coverage_report flag from backend tests job

* Remove third party install from lint check

ankita240796 committed:
Fix part of #6254: Refactor for directives.js and filters.js (#6275)

* Refactor for directives.js

* Remove unused directives

* Refactor for filters.js

* Fix issues

* Add constants

* Move scripts to where they are required

* Add removed service

* Fix failing test

* Fix failing test

apb7 committed:
Run lint tests on Travis temp (#6328)

DubeySandeep commented:
@jeeb0xical, can you please update this PR with upstream/develop so that once it passes the test we can go ahead and merge this?

seanlip committed:
Update mailing list for Travis emails. (#6330)

JeeBoomBoi committed:
Merge branch 'develop' of https://github.com/jeeb0xical/oppia into patch1
